pkg/otlp/config.go

```diff
@@ -78,15 +78,23 @@ func FromAgentConfig(cfg config.Config) (PipelineConfig, error) {
 	if !metricsEnabled && !tracesEnabled {
 		errs = append(errs, fmt.Errorf("at least one OTLP signal needs to be enabled"))
 	}
-
+	logLevel := cfg.GetString(config.OTLPLoggingExporterLogLevel)
+	if logLevel != config.OTLPLoggingExporterLogLevelDisabled &&
+		logLevel != config.OTLPLoggingExporterLogLevelDebug &&
+		logLevel != config.OTLPLoggingExporterLogLevelInfo &&
+		logLevel != config.OTLPLoggingExporterLogLevelWarn &&
+		logLevel != config.OTLPLoggingExporterLogLevelError {
+		errs = append(errs, fmt.Errorf("%s is not a valid value for OTLP logging exporter log level", logLevel))
```

Comment:
@vickenty Thanks for your review! 

> Since this function returns a pipeline object even if there were errors, would it make sense to replace the invalid log level value with the default value? The caller might discard the errors (through a bug or otherwise) and use the pipeline object, and the code that follows probably does not expect invalid values in the pipeline object.

Does this change make sense to you?

```suggestion
		log.Warnf("%s is invalid log level, so OTLP logging exporter will be disabled", logLevel)
		logLevel = config.OTLPLoggingExporterLogLevelDisabled
```

Reply:
Yes, makes sense. Returning an error was fine though, and more in line with how other settings are handled, so maybe still better to return the error instead of logging. The important bit is that we don't put an invalid value into the pipeline config.